feat: update GSoC 2024 home page after org list published (#7118)
jenkins-infra/jenkins.io: content/projects/gsoc/index.adoc

We are participating in Google Summer of Code in 2024. +
// See our link:https://docs.google.com/document/d/1FYOBo12qz24Vxq0TxWuv9ElHH_rHP51ouMsPms4tTmw/edit?usp=sharing[Jenkins GSoC Mentoring Org Application Form].

> We have been accepted by Google to be a mentoring organization in Google Summer of Code 2024.

Reviewer: ```suggestion
Google has accepted us as a mentoring organization in Google Summer of Code 2024.
```

== Contacts

* We use the link:/sigs/gsoc[GSoC SIG] for communications about GSoC.
Projects may also have their own mailing lists, chats, and meetings.

> Please see the details on each of the project pages.

Reviewer: ```suggestion
Please take a look at the details on each of the project pages.
```

You can add the office hours to your calendar when you visit the link:/event-calendar[Jenkins event calendar].
More slots may be added on-demand, e.g. for project-specific discussions.

In addition to these organization-wide meetings, each GSoC project has regular meetings during community bonding and coding phases.

> Please see the project pages for the schedule.

Reviewer: ```suggestion
Please take a look at the project pages for the schedule.
```
